ch32v003fun/ch32v003fun.h

```diff
@@ -2757,6 +2764,7 @@ typedef struct
 #define OB                                      ((OB_TypeDef *)OB_BASE)
 #define ESIG                                    ((ESG_TypeDef *)ESIG_BASE)
 #define EXTEN                                   ((EXTEN_TypeDef *)EXTEN_BASE)
+#define EXTEND                                  ((EXTEND_TypeDef *)EXTEN_BASE)  // Alias to EXTEN
```

Comment:
Typo
- `EXTEN` should be `EXTEND`

Reply:
No, this is what I intended to do.  I am aliasing `EXTEND` to `EXTEN`, but with different register names.